test/fixtures/message-text-html-attachment.eml

```diff
@@ -0,0 +1,57 @@
+Return-Path: <sender@example.com>
+From: sender@example.com
+To: recipient@example.com
+Subject: A message with text, html and a calendar attachment
+Reply-To: sender@example.com
+Mime-Version: 1.0
+Sender: sender@example.com
+Content-Type: multipart/mixed;
+	boundary="_=4g116a4p31245o2q4h634e3y354m3103=_"
+Date: Tue, 27 Apr 2021 12:45:20 +0200
+Message-ID: <3c52f2b1dd3d53eff90acf73500fc058@someclient.example.com>
+
+

```

Comment:
So, it used to add one more newline here?

Reply:
No, in the part below, between the `multipart/alternative` component and the it's first part